ui/i18n/src/main/res/values/strings.xml

```diff
@@ -631,21 +617,38 @@
     <string name="auto_enable_label">Auto-enable</string>
     <string name="sleep_timer_enabled_label">Sleep timer enabled</string>
 
-    <!-- gpodder.net -->
+    <!-- Synchronisation -->
+    <string name="synchronization_choose_title">Choose synchronization provider</string>
+    <string name="synchronization_summary_unchoosen">You can choose from multiple providers to synchronize your subscriptions and episode play state with</string>
+    <string name="dialog_choose_sync_service_title">Choose synchronization provider</string>
+    <string name="gpodnet_description">Gpodder.net is an open-source podcast synchronization service that you can install on your own server. Gpodder.net is independent of the AntennaPod project.</string>
+    <string name="synchronization_summary_nextcloud">Gpoddersync is an open-source Nextcloud app that you can easily install on your own server. The app is independent of the AntennaPod project.</string>
+    <string name="synchronization_host_explanation">Please provide the address (URL or IP address) of your synchronization server.</string>
```

Comment:
* You can use different servers to synchronize with. / You can pick your own server to synchronize with.
* When you have an account on your preferred synchronization server, please provide the server's address here. / When you have identified the synchronization server you will use, please enter its address here. / When you have identified your preferred synchronization server, please enter its address here.

I think 'd go for this:
```suggestion
    <string name="synchronization_host_explanation">You can pick your own server to synchronize with. When you have identified your preferred synchronization server, please enter its address here.</string>
```
@ByteHamster?

Reply:
Sounds good to me